src/main/java/htsjdk/variant/vcf/VCFHeaderLineTranslator.java

```diff
@@ -44,6 +44,7 @@
         mapping.put(VCFHeaderVersion.VCF4_0,new VCF4Parser());
```

Comment:
Can't comment on the line, but `mapping` above can be (and really should be) `final`. It should also be an unmodifiable hash map. e.g.
```
static {
    Map<VcfHeaderVersion, VCFLineParser> map = new HashMap<>();
    map.put(...);
    ...
    mapping = Collections.unmodifiableMap(map);
}

Reply:
Done.